chainer_setup_build.py

```diff
@@ -268,3 +328,164 @@ def get_ext_modules():
 
     check_extensions(extensions)
     return extensions
+
+
+def _nvcc_gencode_options():
+    """Returns NVCC gencode options generated from NVCC command line help."""
+    help_string = subprocess.check_output(
+        ['nvcc', '--help']).decode('ascii').replace('\n', '')
+
+    arch_options = re.findall("'(compute_\d{2})'", help_string)
```

Comment:
I will fix it to accept `\d{2,}`, with making the filter in the following line have numeric comparison, not string one.